src/main/scala/sbtdocker/models.scala

```diff
@@ -97,10 +103,15 @@ case class ImageName(
   override def toString = {
     val registryString = registry.fold("")(_ + "/")
     val namespaceString = namespace.fold("")(_ + "/")
-    val tagString = tag.fold("")(":" + _)
+    val tagString = sanitizedTagString.fold("")(":" + _)
     registryString + namespaceString + repository + tagString
   }
 
+  private def sanitizedTagString = tag.map(s => {
+    val dashesForSlashes = s.replace('/', '-')
```

Comment:
Why is `/` treated differently from all other chars?
With the current logic this will happen: "two words" => "twowords"
Should all invalid chars be converted to `-` perhaps?